src/utils/jsx.ts

```
@@ -0,0 +1,27 @@
+export function createElement(type: string | Function, props: any, ...children: any[]) {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)